lib/tests/modules/disable-module-with-key.nix

```
@@ -0,0 +1,34 @@
+{ lib, ... }:
+let
+  inherit (lib) mkOption types;
+
+  moduleWithKey = {
+    key = __curPos.file + "#moduleWithKey";
```

```suggestion
    key = "disable-module-with-key.nix#moduleWithKey";
```

Let's not rely on an undocumented builtin if we can help it (especially one that's not available under `builtins.*`).

Same for the other module.

---

It's actually not a builtin but a keyword, as a primop wouldn't have enough context, but fair enough.

---

Oh neat, that makes a lot of sense